src/dftbp/dftbplus/inputdata.F90

```diff
@@ -212,6 +212,9 @@ module dftbp_dftbplus_inputdata
     !> Is a perturbation expression in use
     logical :: isDFTBPT = .false.
 
+    !> Is a perturbation expression in use for each geometry step
```

Comment:
I'd suggest to replace it with a more readable alternative, e.g. `doPerturbEachGeom` or similar. (And then also for `isDFTBPT` similarly)

Reply:
Done